src/about/style.js

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from '@material-ui/core'
+
+const root = {
+  margin: '0 auto',
+  maxWidth: '1700px',
+}
+const name = { color: '#89CFF0' }
```

Comment:
Also, please don't forget to extract all these styles into their own separate files :smile: